app/models/collaboration.rb

```diff
@@ -1,4 +1,4 @@
 class Collaboration < ActiveRecord::Base
   belongs_to :user
-  belongs_to :private_wiki 
+  belongs_to :wiki
 end
```

Comment:
This error I overlooked. `Collaboration` must reference the actual model names, `User` (as `:user`) and `Wiki` (as `:wiki`). The `private_wiki` name is defined in app/models/user.rb as an alias to the `belongs_to :wiki` association.